x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.tsx

```diff
@@ -50,6 +51,10 @@ import {
   EffectedPolicySelectProps,
 } from '../../../../components/effected_policy_select';
 
+const StyledEuiHorizontalRule = styled(EuiHorizontalRule)`
```

Comment:
`EuiHorizontalRule` already has `margin` prop - would using that address the need here? They have a `xs` size

Reply:
Ok. I see you are changing only the top, so I'll 👍 . I'm still wondering if it would be best to maybe set the horizontal rule margin to `none` and then use `EuiSpacer` before and after to control the spacing we need.